### PR TITLE
Add macos launcher.

### LINKED ISSUE #5074: Add macOS app to launch PowerShell.
I've been using [this](https://github.com/thezim/launchpowershell) for awhile now. Would be nice if the installer could install something similar making PowerShell launch-able via Spotlight and dock-able.

### PR DESCRIPTION
**Pull request by thezim**
Adds macOS application to launch PowerShell.

Close #5074

### CODE REVIEW

File: tools/packaging/macos/launcher/ROOT/Applications/PowerShell.app/Contents/Info.plist [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	<key>CFBundleExecutable</key>
+	<string>PowerShell.sh</string>
+	<key>CFBundleGetInfoString</key>
+	<string>1.0</string>
+	<key>CFBundleIconFile</key>
+	<string>Powershell</string>
+	<key>CFBundleIdentifier</key>
+	<string>powershell</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
```
Comment (TravisEz13) [inline tools/packaging/macos/launcher/ROOT/Applications/PowerShell.app/Contents/Info.plist]: should we be replacing any of these versions, with the one from the version/release tag?

**Review by TravisEz13 — COMMENTED** [review]
looks good, just one question about the version

File: tools/packaging/macos/launcher/ROOT/Applications/PowerShell.app/Contents/Info.plist (hunk 1 above)
Comment (thezim) [reply tools/packaging/macos/launcher/ROOT/Applications/PowerShell.app/Contents/Info.plist]: That value is related to the plist definition itself and should not change.

https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102088

**Review by TravisEz13 — APPROVED** [review]

**Comment by iSazonov** [comment]
@TravisEz13 Is the PR ready to merge?

**Pull request merged**